### PR TITLE
Quantity limit

### PR DESCRIPTION
**Pull request by LevinGeorge**

### CODE REVIEW

**Comment by danielpaul** [comment]
@LevinGeorge What's the status with this? Would be helpful when you create PR to add relevant tags and comments and assign it to me if I have to do anything.

**Comment by danielpaul** [comment]
@LevinGeorge is this ready to be merged? Is it working?

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    #   redirect_to shop_product_path(@product.shop, @product)
+    # end
+  end
+
+  def edit
+	end
+
+	def update
+		if @review.update(update_comment_params)
+			flash[:success] = "Your comment was successfully updated"
```
Comment (danielpaul) [inline app/controllers/reviews_controller.rb]: You REALLY need to fix the indentation issues @LevinGeorge :P Settings for tab are 2 spaces.

File: app/views/products/_form.html.haml [hunk 2]
```diff
@@ -8,9 +8,9 @@
 			= f.number_field :price, :class => 'text_field', :step => 'any'
 			%br/
 			= f.label :quantity
-			= f.number_field :quantity, :value => 1, :min => 0
+			= f.number_field :quantity, :value => @product.quantity || 1, :min => 0
```
Comment (danielpaul) [inline app/views/products/_form.html.haml]: Why do you need to set the value here manually? Could you not just let the `min` be 1?

File: app/controllers/reviews_controller.rb (hunk 1 above)
Comment (LevinGeorge) [inline app/controllers/reviews_controller.rb]: We tried!! Even Stark can't fix the indentation settings on Atom

File: config/application.yml.template [hunk 3]
```diff
@@ -1,51 +0,0 @@
-# Add configuration values here, as shown below.
```
Comment (danielpaul) [inline config/application.yml.template:1]: Why was this file deleted from git?

File: db/schema.rb [hunk 4]
```diff
@@ -11,11 +11,20 @@
...
 
-ActiveRecord::Schema.define(version: 20160301131158) do
+ActiveRecord::Schema.define(version: 20160303202129) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "comments", force: :cascade do |t|
+    t.integer  "star_rating"
+    t.text     "body"
```
Comment (danielpaul) [inline db/schema.rb]: what's body?

File: db/schema.rb [hunk 5]
```diff
@@ -27,10 +36,21 @@
...
   create_table "purchases", force: :cascade do |t|
-    t.integer  "user_id",    null: false
-    t.integer  "product_id", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.integer  "user_id",             null: false
+    t.integer  "product_id",          null: false
+    t.datetime "created_at",          null: false
+    t.datetime "updated_at",          null: false
+    t.integer  "purchasing_quantity"
```
Comment (danielpaul) [inline db/schema.rb]: could we rename this to just `quantity` ?

File: db/schema.rb [hunk 6]
```diff
@@ -27,10 +36,21 @@
...
+    t.decimal  "purchasing_price"
```
Comment (danielpaul) [inline db/schema.rb]: Rename to `price` ?

File: config/application.yml.template (hunk 3 above)
Comment (LevinGeorge) [inline config/application.yml.template:1]: Just :P Do we need that though? It's just a template

File: db/schema.rb (hunk 4 above)
Comment (LevinGeorge) [inline db/schema.rb]: Body is the actual comment/description.

File: db/schema.rb (hunk 5 above)
Comment (starkshaw) [inline db/schema.rb]: @danielpaul This is the part I did in Payment Integration I don't know why it's here... It's the column that record the quantity and price when purchase is happening.

Comment (danielpaul) [inline db/schema.rb]: okey

Comment (starkshaw) [inline db/schema.rb]: Things so strange. I don't have the migration in current branch, but I do have the columns...

File: db/schema.rb (hunk 4 above)
Comment (danielpaul) [inline db/schema.rb]: could we maybe rename it to `text` or something that would be more intuitive?

File: config/application.yml.template (hunk 3 above)
Comment (danielpaul) [inline config/application.yml.template:1]: Yes but that's the template that other devs need to set up locally. It has all the ENV variables they have to set in `application.yml` file.

File: db/schema.rb (hunk 4 above)
Comment (LevinGeorge) [inline db/schema.rb]: :+1:

File: config/application.yml.template (hunk 3 above)
Comment (danielpaul) [inline config/application.yml.template:1]: So technically everyone should be adding to the template as well dummy values when a new ENV key is introduced.
